server.js

```diff
@@ -149,6 +149,34 @@ async function onRequest(req,res) {
 			}
 		}
 
+		// check if the request url is using "auth-worker.js"
+		// when I just looked for "auth-worker.js" it didn't seem to actually find the url
+		if (req.url == "/js/auth-worker.js") {
+			res.writeHead(200, {
+				...HSTSHeader,
+				"Content-Security-Policy":
+			[
+				`default-src ${[
+					"'self'",
+				].join(" ")};`,
+	
+				`style-src ${[
```

Comment:
I don't think we need to serve the `style-src` policy to a web worker.

Reply:
Yeah, I can see that. I was more cautious and kept the policy the same except fro the `script-src` until it could be looked at by more eyes.